src/server/game/Scripting/ScriptMgr.h

```diff
@@ -839,34 +840,14 @@ class TC_GAME_API GroupScript : public ScriptObject
 
 class TC_GAME_API AreaTriggerEntityScript : public ScriptObject
 {
-    protected:
-
-        AreaTriggerEntityScript(const char* name);
-
-    public:
-        // Called when the AreaTrigger has just been initialized, just before added to map
-        virtual void OnInitialize(AreaTrigger* /*areaTrigger*/) { }
-
-        // Called when the AreaTrigger has just been created
-        virtual void OnCreate(AreaTrigger* /*areaTrigger*/) { }
-
-        // Called on each AreaTrigger update
-        virtual void OnUpdate(AreaTrigger* /*areaTrigger*/, uint32 /*diff*/) { }
-
-        // Called when the AreaTrigger reach splineIndex
-        virtual void OnSplineIndexReached(AreaTrigger* /*areaTrigger*/, int /*splineIndex*/) { }
-
-        // Called when the AreaTrigger reach its destination
-        virtual void OnDestinationReached(AreaTrigger* /*areaTrigger*/) { }
```

Comment:
Well it's pure duplicate and it's not available since a long time, switching to new system only require two copy/paste, i don't think we should keep it